DEVELOPMENTLOG.md

```diff
@@ -1,9 +1,20 @@
-22.09 (???)
+# 22.09 (???)
```

Comment:
Perhaps we should just go 'OpenLoco version 22.10 (???)' here. Little more verbose, but this is for our users.